app/views/api_users/index.html.erb

```diff
@@ -30,7 +30,7 @@
       </tr>
     </thead>
     <tbody>
-      <% @api_users.each do |user| %>
+      <% ApiUser.all.each do |user| %>
```

Comment:
Why this change?  The controller is still populating `@api_users`, and using an `include` to make it more efficient...